src/screens/TopicPageScreen/elements/Empty.js

```diff
@@ -35,7 +36,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontWeight: '400',
     lineHeight: 16,
-    color: '#000000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value has been replaced with a reference to the `COLORS` constant. This change improves consistency and maintainability of the code.
```diff
-    color: '#000000',
+    color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->